otter/util/zkpartitioner.py

```diff
@@ -0,0 +1,133 @@
+"""
+ZooKeeper set-partitioning stuff.
+"""
+
+from twisted.application.internet import TimerService
+from twisted.application.service import MultiService
+from twisted.internet.defer import succeed
+
+
+class Partitioner(MultiService, object):
+    """
+    A Twisted service which uses a Kazoo :obj:`SetPartitioner` to allocate
+    logical ``buckets`` between nodes.
+
+    Multiple servers can instantiate these with the same ``partitioner_path``
+    and ``buckets``, and each server will get a disjoint subset of
+    ``range(buckets)`` allocated to them.
```

Comment:
Thanks for the catch. While developing I had experimented with passing in a number of buckets instead of the actual buckets, but then switched back, and this docstring got lost. thanks :)